src/pages/test-report/components/competency/competency.ts

```diff
@@ -61,14 +65,14 @@ export class CompetencyComponent {
   subscription: Subscription;
 
   isRemoveFaultMode: boolean = false;
-
   faultCount: number;
-
   isSeriousMode: boolean = false;
   hasSeriousFault: boolean = false;
-
   isDangerousMode: boolean = false;
   hasDangerousFault: boolean = false;
+  manoeuvreCompetencyOutcome: ManoeuvreOutcome;
+
+  isManoeuvreCompetency: boolean;
```

Comment:
Due the the problem being very specific and almost non reusable, I would create a separate component which will handle the manoeuvres competencies. This component is already quite complex.

As I'm picking up the serious and dangerous faults I'll make this change, so that this PR can be merged ASAP

Reply:
Might be big refactor to commit to? Not sure I can see huge benefit beyond just making things look nicer?

Reply:
Also I think this is the last bit of complexity to go into competencies now